src/auth0-sso-login.js

```diff
@@ -191,7 +196,15 @@ export default class auth {
         });
       })
       .then(loginInfo => this.getDetailedProfile(loginInfo.idToken, loginInfo.sub))
-      .then(profile => this.profileRefreshed(profile));
+      .then(profile => this.profileRefreshed(profile))
```

Comment:
What if login was successful but profile refresh fails.  I think this then block should be
```javascript
.then(loginInfo => {
   if (loginInfo.idToken && loginInfo.sub) {
     return this.getDetailedProfile(loginInfo.idToken, loginInfo.sub)
       .then(profile => this.profileRefreshed(profile))
       .catch((err) => {
          this.log('Failed to get detailed profile information', err);
        });
     return Promise.resolve();
   }
}, err => {
  this.removeLogin();
  throw err;
});
```

Reply:
Good point. Fixed.

I also moved the removeLogin away from the main procedure, and it handles all types of promise rejections now in the catch block.